test/luger_test.erl

```diff
@@ -381,8 +382,8 @@ throttling_test_() ->
       ?T("throttling",
          begin
              [luger:alert("throttling.main", "msg") || _ <- lists:seq(0, 100)],
-             ?assert_log_count(Logger, stderr, 5, 10),
-             ?assert_log_count(Logger, statsd, 5, 10),
+             ?assert_log_count(Logger, stderr, 3, 3),
+             ?assert_log_count(Logger, statsd, 3, 3),
```

Comment:
You seem to have tightened up the bounds on the expected message count but I'm not seeing any code that would ensure that all message are sent and received within the same second. Unless I missed something, I would expect this to fail randomly.

Reply:
It hasn't in the dozen-or-so times I've run the tests. If you prefer, I could undo this change, but I think a more proper solution would be to have `meck` mock `erlang:monotonic_time` and control the advancement of time from within the tests.

That might be for another day, though.

Reply:
After having talked in person, the rest of the tests use meck so there's not real reason to be picky about it for this use case. So yeah, go ahead and meck time to get exact results but also change the `assert_log_count` to no longer be a range as it will no longer be necessary.